drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/factory/remote/reusable/TestRemoteWebDriverFactorySessionStoring.java

```diff
@@ -75,8 +75,9 @@
     private InitializationParameter initializationParameter;
 
     @BeforeClass
-    public static void skipIfEdgeBrowser() {
-        Assume.assumeFalse(System.getProperty("browser", "phantomjs").equals("edge"));
+    public static void skipIfEdgeOrChromeHeadlessBrowser() {
+        String browser = System.getProperty("browser", "phantomjs");
+        Assume.assumeFalse(browser.equals("edge") || browser.equals("chromeHeadless") || browser.equals("chromeheadless"));
```

Comment:
why do we skip it? in this case, it should work too... You just need check when the chromeheadless is used and in then set appropriate ChromeOption argument